
Some user-space apps (e.g. reiserfsprogs) do need the unaligned macros, ...

  - Rene Rebe <rene@exactcode.de>

--- ./include/asm-ppc/unaligned.h.orig	2004-10-25 12:12:02.331864008 +0200
+++ ./include/asm-ppc/unaligned.h	2004-10-25 12:12:09.347797424 +0200
@@ -1,4 +1,4 @@
-#ifdef __KERNEL__
+
 #ifndef __PPC_UNALIGNED_H
 #define __PPC_UNALIGNED_H
 
@@ -15,4 +15,4 @@
 #define put_unaligned(val, ptr) ((void)( *(ptr) = (val) ))
 
 #endif
-#endif /* __KERNEL__ */
+
